example/cursors.ts

```diff
@@ -0,0 +1,39 @@
+import { EditorView } from '@codemirror/view'
```

Comment:
I guess it can be published as a package in order to be used regardless of themes.